tests/testsuite/conda.rs

```diff
@@ -2,12 +2,15 @@ use ansi_term::Color;
 use std::io;
 
 use crate::common;
+use crate::common::TestCommand;
+use toml::Value;
```

Comment:
This isn't used so can be removed to make Clippy happy.

```suggestion
```

Reply:
Thanks for catching this, done.